pkg/stomp/StompConnectionFactory.php

```diff
@@ -74,14 +74,14 @@ public function __construct($config = 'stomp:')
      */
     public function createContext(): Context
     {
-        if ($this->config['lazy']) {
-            return new StompContext(
-                function () { return $this->establishConnection(); },
-                $this->config['target']
-            );
-        }
+        $stomp = $this->config['lazy']
+            ? function () { return $this->establishConnection(); }
+        : $this->establishConnection();
```

Comment:
Normally I indent lines like `79`, but the code style suggested this, apologies!